File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusGameChallengeManageForm.kt
package com.sdkwork.backend

data class PlusGameChallengeManageForm(
    val status: String? = null,
    val winnerUserId: Int? = null,
    val roomId: Int? = null,
    val matchId: Int? = null,
    val reason: String? = null
)

File: sdkwork-backend-sdk-typescript/src/types/plus-game-challenge-manage-form.ts
/** 棋牌游戏挑战赛管理表单 */
export interface PlusGameChallengeManageForm {
  /** 挑战状态 */
  status?: 'PENDING' | 'ACCEPTED' | 'REJECTED' | 'CANCELED' | 'EXPIRED' | 'SETTLED';
  /** 获胜方用户ID，仅结算时使用 */
  winnerUserId?: number;
  /** 关联房间ID */
  roomId?: number;
  /** 关联对局ID */
  matchId?: number;
  /** 处理备注 */
  reason?: string;
}
